src/foamadapter/solver/pimplefoam.py

```diff
@@ -0,0 +1,144 @@
+
+import pybFoam
+from pybFoam import (
+    Info,
+    Time,
+    Word,
+    adjustPhi,
+    computeCFLNumber,
+    constrainHbyA,
+    constrainPressure,
+    createPhi,
+    dictionary,
+    fvc,
+    fvm,
+    fvMesh,
+    fvScalarMatrix,
+    fvVectorMatrix,
+    pimpleControl,
+    setRefCell,
+    solve,
+    surfaceScalarField,
+    volScalarField,
+    volVectorField,
+)
+from pybFoam.turbulence import incompressibleTurbulenceModel, singlePhaseTransportModel
+
+
+class CFLNumber:
+
+    def __init__(self):
+        self.criteria = []
+        self.GREAT = 1e30
+        self.SMALL = 1e-15
+        self.maxDeltaT = 1e-3  # Default maximum deltaT
+
+    def setDelta(self, runTime, phi):
+        deltaT = runTime.deltaTValue()
+
+        if not self.criteria:
+            return
+
+        max_cfl_number, mean_cfl_number = computeCFLNumber(phi)
```

Comment:
As far as I can see in the rest of code camelCase is used.
```suggestion
        maxCFLNumber, meanCFLNumber = computeCFLNumber(phi)
```